test/src/main/java/org/axonframework/test/aggregate/AggregateTestFixture.java

```diff
@@ -480,6 +492,7 @@ private void registerAggregateCommandHandlers() {
         if (!explicitCommandHandlersSet) {
             AggregateAnnotationCommandHandler.Builder<T> builder = AggregateAnnotationCommandHandler.<T>builder()
                     .aggregateType(aggregateType)
+                    .aggregateModel(aggregateModel())
```

Comment:
If a model is provided, is registering the aggregateType still necessary?

Reply:
Apparently yes, but fixing it does not belong to this issue :)